author.tf

```diff
@@ -0,0 +1,436 @@
+module "author-eq-ecs" {
+  source                   = "github.com/ONSdigital/eq-terraform-ecs?ref=319-create-terraform-script-for-author"
```

Comment:
This is now merged so we don't have to point at the branch anymore?

Reply:
Do we need to tag it? Or should we just use `master`?